### PR TITLE
Add state and locality

### PR DESCRIPTION
**Pull request by mbonness**
Our registration authority requires us to submit state and locality in our CSR.  These changes add the ability to set a state or province name and locality.

### CODE REVIEW

File: CertificateSigningRequestSwift/CertificateSigningRequestSwift.swift [hunk 1]
```diff
@@ -34,37 +34,43 @@ import Foundation
 public class CertificateSigningRequest:NSObject {
     private let OBJECT_commonName:[UInt8] = [0x06, 0x03, 0x55, 0x04, 0x03]
     private let OBJECT_countryName:[UInt8] = [0x06, 0x03, 0x55, 0x04, 0x06]
+    private let OBJECT_stateOrProvinceName:[UInt8] = [0x06, 0x03, 0x55, 0x04, 0x08]
```
Comment (wuf810) [inline CertificateSigningRequestSwift/CertificateSigningRequestSwift.swift]: Just my personal preference but I would put these properties in alphabetical order (to match the order of the current)

File: CertificateSigningRequestSwift/CertificateSigningRequestSwift.swift [hunk 2]
```diff
@@ -34,37 +34,43 @@ import Foundation
...
+    private let OBJECT_localityName:[UInt8] = [0x06, 0x03, 0x55, 0x04, 0x07]
     private let OBJECT_organizationName:[UInt8] = [0x06, 0x03, 0x55, 0x04, 0x0A]
     private let OBJECT_organizationalUnitName:[UInt8] = [0x06, 0x03, 0x55, 0x04, 0x0B]
     private let SEQUENCE_tag:UInt8 = 0x30
     private let SET_tag:UInt8 = 0x31
     
     private let countryName:String?
+    private let stateOrProvinceName:String?
```
Comment (wuf810) [inline CertificateSigningRequestSwift/CertificateSigningRequestSwift.swift]: Same again; alphabetical order

File: CertificateSigningRequestSwift/CertificateSigningRequestSwift.swift (hunk 1 above)
Comment (mbonness) [reply CertificateSigningRequestSwift/CertificateSigningRequestSwift.swift]: Done.

File: CertificateSigningRequestSwift/CertificateSigningRequestSwift.swift (hunk 2 above)
Comment (mbonness) [reply CertificateSigningRequestSwift/CertificateSigningRequestSwift.swift]: Also done.

**Pull request merged**